src/consent-manager.tsx

```diff
@@ -22,11 +23,12 @@ export const injectConsentManagerApp = async (
 ): Promise<ConsentManagerAPI> => {
   // The interface hasn't initialized yet
   if (consentManagerAPI === null) {
+    const mergedConfig = getMergedConfig();
```

Comment:
You don't have to use getMergedConfig() here. That function is for extracting some rich data structures out of the config, wheras this is a simple settings.flagName string flag

Reply:
That's true, but just using settings precludes the user from overriding it with the self.transcend.consentManagerConfig method. If we still want the user to be able to do that we would have to reference both settings and extraConfig, at which point I think it's close enough to the merged config it would warrant pulling it in here. Not to mention that the merged config was being retrieved already in Main.tsx - all I did was move it one component up - so there's not really any extra computation being performed from the change here.

If we don't care about the extraConfig override I can definitely make the change to just using settings, but this seemed like a decent syntactic shortcut to accessing both of those (and potentially the base config if we end up putting the default there in a future refactor or something)

Reply:
ah ok that makes sense if we want to support that. I don't have any strong preference here